bigbluebutton-html5/imports/api/group-chat-msg/index.js

```diff
@@ -6,15 +6,17 @@ const collectionOptions = Meteor.isClient ? {
 
 const GroupChatMsg = new Mongo.Collection('group-chat-msg');
 const UsersTyping = new Mongo.Collection('users-typing', collectionOptions);
+const Reactions = new Mongo.Collection('msg-reactions');
```

Comment:
I suggest to call this collection as `group-chat-msg-reaction`. In order to keep the list of collections clearer.